devices/power.go

```
@@ -33,28 +42,26 @@ func amtPowerWrite(device *sdk.Device, data *sdk.WriteData) error {
 	// When writing to a AMT Power device, we always expect there to be
 	// raw data specified. If there isn't, we return an error.
 	if len(raw) == 0 {
-		return fmt.Errorf("no values specified for 'raw', but required")
+		logger.Error("no values specified for 'raw', but required")
+		return nil
```

should revert this -- we want to return the error, not return nil